software/CONFIGURATION.md

```diff
@@ -41,8 +40,8 @@ default configuration:
 
 # Experimental configuration
 
-Other configuration properties are used by [experimental features](software/firmware/experimental/__init__.py)
-and can be set using a similar static configuration file. This file is located at `/config/config_ExperimentalConfig.json` 
+Other configuration properties are used by [experimental features](/software/firmware/experimental/experimental_config.py)
```

Comment:
I wasn't sure why it was pointing to the `__init__` file initially; I figured it would make more sense to link to the file that actually implements the experimental configuration that's being described.

Reply:
The `__init__.py` file contains documentation on the experimental module itself.

Reply:
Reverted